src/main/resources/schema-postgres.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE IF EXISTS users;
+CREATE TABLE public.users(
```

Comment:
Please create index on this table with the field we use most for inquiring this table. 

Users login with email address, so, I believe we should have email address as index.